modules/ossm-deploying-bookinfo-application.adoc

```diff
@@ -24,12 +24,17 @@ The *Display name* and *Description* fields provide supplementary information an
 
 . Click *Create*.
 
-. Apply the the {Istio} injection label to the `bookinfo` namespace by entering the following command at the command line:
+. Apply the {Istio} injection label to the `bookinfo` namespace by entering the following command at the CLI:
 +
 [source,terminal]
 ----
-$ oc label namespace bookinfo istio.io/rev=<revision-name>
+$ oc label namespace default istio-injection=enabled
```

Comment:
@rh-tokeefe Thanks for adding the note. You also have to update the namespace to "bookinfo"

i.e., it should read
$ oc label namespace bookinfo istio-injection=enabled

Reply:
This is correct, I think I made a mistake copy/pasting the command into the issue.

Reply:
Just pushed the update.